### PR TITLE
fix err information of certutil

### PR DESCRIPTION
**Pull request by muxuelan**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
If not add the modify, err equals nil here, and this will affect the generation of certificate.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/cloudhub/servers/httpserver/certutil.go [hunk 1]
```diff
@@ -100,10 +101,12 @@ func NewCertFromCa(cfg *certutil.Config, caCert *x509.Certificate, serverKey cry
 	}
 	if len(cfg.CommonName) == 0 {
 		fmt.Println("must specify a CommonName")
+		err :=errors.New("must specify a CommonName")
 		return nil, err
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/servers/httpserver/certutil.go:105]:
```suggestion
		return nil, errors.New("must specify a CommonName")
```

File: cloud/pkg/cloudhub/servers/httpserver/certutil.go [hunk 2]
```diff
@@ -100,10 +101,12 @@ func NewCertFromCa(cfg *certutil.Config, caCert *x509.Certificate, serverKey cry
...
 	}
 	if len(cfg.Usages) == 0 {
 		fmt.Println("must specify at least one ExtKeyUsage")
+		err :=errors.New("must specify at least one ExtKeyUsage")
 		return nil, err
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/servers/httpserver/certutil.go:110]:
```suggestion
		return nil, errors.New("must specify at least one ExtKeyUsage")
```

File: cloud/pkg/cloudhub/servers/httpserver/certutil.go (hunk 1 above)
Comment (fisherxu) [reply cloud/pkg/cloudhub/servers/httpserver/certutil.go:105]: fmt print can be deleted here.

**Review by fisherxu — COMMENTED** [review]
Please also squash the commits.

File: cloud/pkg/cloudhub/servers/httpserver/certutil.go (hunk 1 above)
Comment (muxuelan) [reply cloud/pkg/cloudhub/servers/httpserver/certutil.go:105]: Okay ,I will ,thanks.

**Pull request merged**